build.sbt

```diff
@@ -141,18 +139,20 @@ lazy val sharedSettings = MimaSettings.settings ++ scalaVersionSettings ++ Seq(
 
   pomIncludeRepository := { _ => false },
 
-  pomExtra := {
-    <scm>
-      <url>https://github.com/typelevel/scalacheck</url>
-      <connection>scm:git:git@github.com:typelevel/scalacheck.git</connection>
-    </scm>
-    <developers>
-      <developer>
-        <id>rickynils</id>
-        <name>Rickard Nilsson</name>
-      </developer>
-    </developers>
-  }
+  scmInfo := Some(
+    ScmInfo(
+      url("https://github.com/typelevel/scalacheck"),
+      "scm:git:git@github.com:typelevel/scalacheck.git"
+    )
+  ),
+  developers := List(
+    Developer(
+      id    = "rickynils",
+      name  = "Rickard Nilsson",
+      email = "rickynils@gmail.com",
+      url   = url("https://github.com/rickynils")
+    )
+  )
```

Comment:
We should probably add ourselves to the developers list at this point.

Reply:
That's fine.  I'd rather add ourselves in a follow-up, though.

Reply:
👍